src/source/CppGenerator.scala

```diff
@@ -80,7 +80,7 @@ class CppGenerator(spec: Spec) extends Generator(spec) {
           w.w(s"constexpr $self operator$op($self lhs, $self rhs) noexcept").braced {
             w.wl(s"return static_cast<$self>(static_cast<$flagsType>(lhs) $op static_cast<$flagsType>(rhs));")
           }
-          w.w(s"constexpr $self& operator$op=($self& lhs, $self rhs) noexcept").braced {
+          w.w(s"inline $self& operator$op=($self& lhs, const $self rhs) noexcept").braced {
```

Comment:
I didn't previously notice the `const` here, and it doesn't appear to show up in the generated code below.  Was this added late?

If I'm understanding this correctly, the `const` here seems unnecessary since flags are passed by value anyway, so it's meaningless to the caller.  I prefer to avoid passing/returning by "const value" since it clutters the interface unnecessarily.